src/coll/mod.rs

```diff
@@ -240,34 +252,23 @@ impl Collection {
 
     // Helper method for all findAndModify commands.
     fn find_and_modify(&self,
-                       cmd: &mut bson::Document,
                        filter: bson::Document,
+                       options: bson::Document,
                        _max_time_ms: Option<i64>,
```

Comment:
I've noticed we don't seem to actually use this option anywhere in the driver. I think it might be a good idea to look at how other drivers use it, since it probably is included in the spec for *something*.